landing_page/urls.py

```diff
@@ -5,7 +5,9 @@
 
 urlpatterns = [
     path("", views.index, name="index"),
+    path("get-csrf-token/", views.get_csrf_token, name="get_csrf_token"),
```

Comment:
why do we need a csrf token? realizing i use it in mine, but not sure why we need to call for it 
![image](https://github.com/jonathanhouge/ReeldIn/assets/111781413/7c0f1256-9301-4059-b591-20ad691dded7)

Reply:
okay, maybe mine like, gets it intrinsically as it's a wrapper or something? i'm not 100%

anyway, i am getting a lot of errors here about how something is wrong
![image](https://github.com/jonathanhouge/ReeldIn/assets/111781413/08bd1ccb-3586-44ee-a221-e40797e408ab)

Reply:
I don't know why I need it but I was getting forbidden errors saying that the token was missing. I found online that I should have been able to include it in the base.html but that wasn't working which is why I added the route. I might be able to put it inside the div like you have it in your form though

Reply:
Those errors are from the search bar right?

Reply:
> Those errors are from the search bar right?

yeah as i am filling out the search bar, this is happening... so maybe it isn't waiting for that token?

Reply:
That's weird because I don't get any errors on mine. And everything is async so I don't know how its happening

Reply:
Looks like it was resolved but I thought it'd be worth mentioning I didn't get any errors